src/video-player.js

```diff
@@ -24,6 +25,7 @@ import EmbedPlayer from './embed-player.js';
             assetId,
             ...tokenParameter,
             ...posterImageUrlParameter,
+            autoplay,
```

Comment:
Add the parameter's value handler. Otherwise, `?autoplay=true/false` will always cause the autoplay